docs/faq.rst

```diff
@@ -709,6 +709,31 @@ Send a private email to the :ref:`security contact <security-contact>`
 if you think you have discovered a security issue.
 Please disclose security issues responsibly.
 
+How important is path/to/repo/nonce?
+------------------------------------
+
+Borg uses :ref:`AES-CTR encryption <borg_security_critique>`. An
+essential part of AES-CTR is a sequential counter that must **never**
+repeat. If the same value of the counter is used twice in the same repository,
+an attacker can decrypt the data. The counter is stored in the home directory
+of each user (under $HOME/.config/borg/security) as well as in the repository. When creating a new archive borg uses
```

Comment:
add /nonce to the path so we have the complete filename.

Reply:
maybe we could have the headline as just "how important are the nonce files" and then specify both full paths on the section below.

Reply:
done